generate-audio.py

```diff
@@ -61,6 +67,33 @@ def load_captions(config):
         raise Exception('Unsupported subtitles format')
 
 
+# TODO figure out better way of defining break length
+def define_break(diff_length, num_of_pauses):
+    length_of_pause = diff_length / num_of_pauses
+    if diff_length / num_of_pauses > 2:
+        return 1000
+    elif 1 < length_of_pause < 2:
+        return 800
+    else:
+        return 500
+
+
+def extend_sentence_audio(sentence_audio, caption):
+    audio_duration = sentence_audio.duration_seconds
+    caption_start_time = caption_start(caption)
+    caption_end_time = caption_end(caption)
+    diff = ((caption_end_time - caption_start_time) - audio_duration).__round__(3)
```

Comment:
można to lekko zoptymalizowąc. Jeśli `diff` < noticeable_difference (np: 0,25) to zaakceptuj bieżące audio